src/Helpers/XmlFailureMitigation.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\BankFiles\Helpers;
+
+/**
+ * A helper class that works to mitigate XML-related issues.
+ */
+class XmlFailureMitigation
+{
+    /**
+     * Attempts to work around common data inconsistencies that the banks find acceptable (for some odd reason).
+     *
+     * @param string $content The XML content.
+     *
+     * @return string
+     */
+    public static function tryMitigateParseFailures(string $content): string
+    {
+        // Split the content in to individual lines
+        $lines = \explode(\PHP_EOL, \trim($content));
+
+        // Iterate through each line from the content string
+        foreach ($lines as &$line) {
+            // Find any matching node element name and content in the line
+            $result = \preg_match_all('/<([A-Za-z0-9-_]+)?(?:[^\>]+)?>(.*)<\/\1>/', $line, $matches, \PREG_SET_ORDER);
+            if ($result === false || $result === 0) {
+                continue;
+            }
+
+            // Begin to iterate through each node key and value
+            // We assume here that there *could* be more than one XML element in a given line.
+            foreach ($matches as $match) {
+                // If the match does not contain three elements, skip
+                if (\count($match) !== 3) {
+                    // @codeCoverageIgnoreStart
+                    // Sanity check only and unable to be tested.
+                    continue;
```

Comment:
instead of continuing in these sanity checks we should throw an ImpossibleException - we dont want to silently fail on a situation that could be world ending (ie, preg_match_all not working properly)